bin/summarizedexperiment.r

```diff
@@ -5,15 +5,15 @@ library(SummarizedExperiment)
 ## Create SummarizedExperiment (se) object from Salmon counts
 
 args <- commandArgs(trailingOnly = TRUE)
-if (length(args) < 2) {
-    stop("Usage: salmon_se.r <coldata> <counts> <tpm>", call. = FALSE)
+if (length(args) < 3) {
+    stop("Usage: salmon_se.r <coldata> <counts> <tpm> <tx2gene>", call. = FALSE)
```

Comment:
Should we still be referring to `salmon_se.r` here? Might be worth checking the whole script for mentions of salmon.